docs/docusaurus.config.js

```diff
@@ -43,8 +43,7 @@ module.exports = {
           items: [
             {
               label: 'Backend Docs',
-              href:
-                'https://github.com/anitab-org/anitab-forms-backend/wiki',
+              href: 'https://github.com/anitab-org/anitab-forms-backend/wiki',
```

Comment:
Don't make changes unrelated to this PR.

Reply:
I just ran ``` npm run format ```. It's only formatting change..  Should I remove it ?

Reply:
Please update your dependency and reformat the code.